pages/bare_metal_cloud/dedicated_servers/api-os-installation/guide.en-gb.md

```diff
@@ -164,123 +164,42 @@ Each question has the following attributes:
 
 ¹ If a mandatory question without default value is not answered, the API will return an error.
 
-### Disk Groups <a name="disk-group"></a>
-
-Some [dedicated servers](/links/bare-metal/bare-metal) have multiple groups of disks. For example, one group with SATA disks and another group with SSD disks. Those servers are sometimes also called **hybrid servers**.
-
-To list the disk groups and their disks, you can use the following API call in order to identify the disk group on which you want the OS to be installed:
-
-> [!api]
->
-> @api {v1} /dedicated/server GET /dedicated/server/{serviceName}/specifications/hardware
->
-
-Example reply:
-
-```json
-{
-  "bootMode": "uefi",
-  "defaultHardwareRaidSize": null,
-  "description": "SCALE-7 - AMD Epyc 7763",
-  "expansionCards": null,
-  "threadsPerProcessor": 128,
-  "memorySize": {
-    "unit": "MB",
-    "value": 524288
-  },
-  "defaultHardwareRaidType": null,
-  "diskGroups": [
-    {
-      "defaultHardwareRaidType": null,
-      "numberOfDisks": 2,
-      "diskGroupId": 1,
-      "diskType": "SSD",
-      "defaultHardwareRaidSize": null,
-      "diskSize": {
-        "unit": "GB",
-        "value": 480
-      },
-      "description": "2 X Disk SSD 480 GB, JBOD",
-      "raidController": null
-    },
-    {
-      "defaultHardwareRaidType": null,
-      "numberOfDisks": 2,
-      "diskGroupId": 2,
-      "diskType": "NVME",
-      "defaultHardwareRaidSize": null,
-      "diskSize": {
-        "unit": "GB",
-        "value": 1920
-      },
-      "raidController": null,
-      "description": "2 X Disk NVME 1920 GB, JBOD"
-    }
-  ],
-  "processorArchitecture": "x86_64",
-  "coresPerProcessor": 64,
-  "processorName": "Epyc7763",
-  "formFactor": "1u",
-  "motherboard": "S8036-M",
-  "numberOfProcessors": 1,
-  "usbKeys": null
-}
-```
-
-In this example, there are 2 disk groups:
-
-- the first one (diskGroupId=1) contains 2 x 480 GB disks.
-- the second one (diskGroupId=2) contains 2 x 1,9 TB disks.
-
-> [!primary]
->
-> By default, the OS will be installed on diskGroupId 1.
->
-
 ### Create an OS installation task <a name="install-task"></a>
 
 When you have gathered all the information that you need, you can create the OS installation task with the following API call:
 
 > [!api]
 >
-> @api {v1} /dedicated/server POST /dedicated/server/{serviceName}/install/start
+> @api {v1} /dedicated/server POST /dedicated/server/{serviceName}/reinstall
 >
 
 With the following parameters:
 
 |Parameter|Value|Required|
 |---|---|---|
 |serviceName|Server name|✅|
-|templateName|OS to install|✅|
-|diskGroupId|Id of the disk group on which the OS will be installed|❌|
-|customHostname|Custom hostname|❌|
-|userMetadata|Answers to the OS specific questions|❌|
+|operatingSystem|OS to install|✅|
+|customizations|OS reinstallation customizations|⚠️¹|
+|storage|OS reinstallation storage configurations|❌|
+|properties|Arbitrary properties to pass to cloud-init's config drive datasource|❌|
```

Comment:
```suggestion
|properties|Arbitrary properties to transfer to cloud-init's config drive datasource|❌|
```

Reply:
This is what has been written in API:
https://eu.api.ovh.com/console/?section=%2Fdedicated%2Fserver&branch=v1#post-/dedicated/server/-serviceName-/reinstall (currently this route is only available from an OVHcloud IP)